### ISSUE TITLE
Problems with accuracy after upgrading Optikey 3.2.0

### ISSUE THREAD

**Issue by lsaranto** [OptiKey/OptiKey#808]
This is a bit vague, but I have a feeling that eye-tracking (Tobii 4C)  is not as responsive or accurate as it was before upgrading Optikey pro from 3.1.0 to 3.2.0. It's like the fixations don't want to lock on the key, but at the same time, the fixations might hang on a key and don't always want to jump to the next one over. I've had accuracy issues before if I'm not facing the monitor perfectly, but then I have always been able to find a spot where to look a bit off-center of a key and then it works smoothly. But now it works smooth, and then a key suddenly acts out.

I scanned through the release notes, but I didn't notice any mentions which might be related to what I'm experiencing. I tried recalibrating a couple of times, but it didn't really help. I did run into another issue that Tobii software dies after recalibration returns and the Tobii service needs to be killed and restarted.

I also tried to compare the setting files but I didn't notice any differences.

I didn't yet try downgrading because I wasn't sure if I need to do any extra steps.

**Comment by AdamRoden** [comment]
@lsaranto 
First, can you confirm that you are not running a 3.2 pre-release? 
Second, what do you have here:
![image](https://user-images.githubusercontent.com/45572061/157959760-6748b3f8-766a-424b-b029-9c429f9e70d7.png)

**Comment by lsaranto** [comment]
@AdamRoden It's the second day and 3.2 is definitely behaving differently than 3.1.

 It's not pre-release. I downloaded it only after I got an update notification in Optikey.

The first two settings after source are medium, no checkmark in the checkbox, and 150 ms in the last one. These are the default values I believe. I don't remember playing around with them. I have many times thought about them. Some suggestions in the wiki on what to try might be good.

I see you have Tobii 5. Do you think it's worth the upgrade?

**Comment by AdamRoden** [comment]
@lsaranto The first setting is how much Tobii's device filters your gaze before providing the data to Optikey. The second is how much Optikey averages out the continuos flow of gaze data. The checkbox applies the averaging over larger movements. 
Try low, low, checked and some other combinations.
The Tobii 5 for gaming is not worth an upgrade, but the Tobii PCEye 5 ($1200+) does offer better, though still unreliable, tracking in bright light.

**Comment by lsaranto** [comment]
@AdamRoden I tried low, low, check and it did help. For fun, I then tried going back to med, med, no check and it's still working relatively well. The worst stickiness is still gone. So I'm not sure what actually happened. Maybe I remembered wrong what the settings have been. Or maybe the settings in effect somehow didn't match with the values showing. I'm alright with the current performance. I'll see if I still try going back to 3.1.0 out of curiosity

Thank you!

**Comment by JuliusSweetland** [comment]
@lsaranto Good news. It may be that the default settings changed between 3.1 and 3.2 (3.2 took so long to put together that I can't remember every change that's in it!). The result would be a relatively large change in "feel" if the processing amounts had changed.

@AdamRoden Thank you for your help here.
